knowledge-base/telerik-reporting-toolbox-items-are-missing.md

```diff
@@ -125,9 +125,33 @@ The cause of the issues is RIA services. Uninstall WCF RIA Services and WCF RIA
  HKEY\_LOCAL\_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Silverlight\v4.0\AssemblyFoldersEx\Ria Services v1.0 Silverlight Libraries\Toolbox 
  
  HKEY\_LOCAL\_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Silverlight\v5.0\AssemblyFoldersEx\Ria Services v1.0 Silverlight Libraries\Toolbox
+ 
+## Reset of the Visual Studio Toolbox 
+
+If the above doesn't help, you may reset completely the Visual Studio Toolbox, which is done as follows:
+
+1. Close all Visual Studio instances. 
+
+2. Open the **%localappdata%\Microsoft\VisualStudio\16.0_[InstanceID]**  folder and remove all __*.tbd__ files. 
+
+3. Open the **%localappdata%\Microsoft\VisualStudio\16.0_[InstanceID]**  folder and delete the __ComponentModelCache__ folder. 
+
+4. Load the Visual Studio __privateregistry.bin__ using these steps: 
+
+   - Launch __RegEdit.exe__ 
```

Comment:
Maybe insert a line between each row so that the text doesn't get too small? Not sure if it will work though in a nested scenario like this.

Reply:
I think it will be OK